src/main/scala/ai/privado/tagger/source/IdentifierTagger.scala

```diff
@@ -21,7 +21,7 @@
  *
  */
 
-package ai.privado.languageEngine.ruby.tagger.source
+package ai.privado.tagger.source
```

Comment:
Did we only replaced the package name, ideally we should also do a refactor to place this in the appropriate package as well

Reply:
Yes, I've placed it in the appropriate package and GH is indicating that the file has been moved to a new location. Or am I mistaken here?